### PR TITLE
Map gr key with go to references

### PR DESCRIPTION
**Pull request by ElieCaratge**

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -156,6 +159,7 @@ nnoremap aw vaw
 nmap <silent> gs :call CocAction('jumpDefinition', 'split')<CR>
 nmap <silent> gd :call CocAction('jumpDefinition', 'vsplit')<CR>
 nmap <silent> gt :call CocAction('jumpDefinition', 'tabe')<CR>
+nmap <silent> gr <Plug>(coc-references)
```
Comment (annepinson) [inline vimrc:162]: Guidelines

Comment (annepinson) [reply vimrc:162]: Should be referenced in README

Comment (annepinson) [reply vimrc:162]: Buuut ok it's approved :)

Comment (ElieCaratge) [reply vimrc:162]: All right, done :smirk:

**Pull request merged**